jobs/config.json

```diff
@@ -8899,7 +8899,7 @@
       "--cluster=",
       "--deployment=kubernetes-anywhere",
       "--env-file=jobs/platform/gce.env",
-      "--extract=ci/latest-1.10",
+      "--extract=ci/latest-bazel-1.10",
       "--gcp-zone=us-central1-f",
       "--kubeadm=ci",
       "--kubernetes-anywhere-kubelet-ci-version=latest-1.10",
```

Comment:
Going to make an educated guess that you are trying to install a ci version that is now has bazel in its name. If so, you will also want to update the --kubernetes-anywhere-kubernetes-version flag to also include bazel in its name.